proteus/mprans/MCorr.h

```diff
@@ -510,7 +560,28 @@ namespace proteus
       r = porosity*smoothedHeaviside(epsHeaviside,phi+u) - H;
       dr = porosity*smoothedDirac(epsDirac,phi+u);
     }
-
+    inline
+    void evaluateCoefficients_explicit_comp_quad(const double& epsHeaviside,
+    			      const double& epsDirac,
+    			      const double& phi,
+    			      const double& H,
+			      const double& H_old,///YY
+				  const double v[nSpace],///YY
+    			      const double& u,
+    			      const double& porosity,
+    			      double& r,
+    			      double& dr,
+				  double f[nSpace],
+				  double df[nSpace])
+    {
+          r = porosity*smoothedHeaviside(epsHeaviside,phi+u) - H_old;
+          dr = porosity*smoothedDirac(epsDirac,phi+u);
+          for (int I=0; I < nSpace; I++)
+          {
+        	  	  f[I] = v[I]*porosity*u;
```

Comment:
The explicit flux is `porosity*smoothedHeaviside(epsHeaviside,phi_old)`. Remember that here, `u` is just the correction to  the signed distance (`phi`).